tbot/machine/linux/special.py

```diff
@@ -50,13 +50,27 @@ class _Stdio(Special[H]):
     def _redir_token(self) -> str:
         raise tbot.error.AbstractMethodError()
 
+    @property
+    def _both(self) -> bool:
+        return False
```

Comment:
```python
_both = False
```

is equivalent for this case.  The `@property` dance is only needed when the attribute should be mandatory to overwrite (and thus the `@abc.abstractmethod` decorator needs to be placed on it).